docs/fundamentals/week-2/session.md

```diff
@@ -145,7 +145,41 @@ Read ahead to Ship It to see where to paste your code.
 :::note Exercise: Click! (25m)
 In the same pairs, try to implement one user story. While implementing, try to break down into steps where you can visually check that your code runs as expected.
 
-:::
+---
+
+## Professional development
+
+::: Transparency is the key to success! 
+
+A key part of being a successful developer is the ability to give and receive feedback, in order to start that process you must be able to showcase your work in progress. Now that you have had a chance to try some coding, it’s time to be transparent and talk about it. 
+
+Find a partner! 
+
+**20 minutes: **
+
+In pairs take turns to ask the questions below:  
+When it comes to showing your work in progress, what are your three biggest fears?
+When it comes to showing your work in progress, how will you overcome your three biggest fears? 
+When it comes to showing your work in progress, what’s the absolute worst thing that can happen? 
+When it comes to showing your work in progress, what is the best thing that can happen? 
+When it comes to the working world, how will you support your team mates?
+When it comes to the working world, what do you need from your team mates? 
```

Comment:
I would worry that the first question is a bit hard to answer without context.

I wonder if we should swap these around and frame it as "what do you need, what will you do to support your team mates needs" would help with that?

Reply:
That's a great suggestion! 

Let's make that change

Reply:
Great! 🙂 Do you need a hand with doing that? (I'm afraid I'm unlikely to have much time this week to help but I'm sure you could ask one of the other syllabus team members)

Reply:
Thanks Ali! All done :)